db/migrate/20211008032520_add_profile_to_participant_declarations.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+class AddProfileToParticipantDeclarations < ActiveRecord::Migration[6.1]
+  disable_ddl_transaction!
+
+  def change
+    add_reference :participant_declarations, :participant_profile, index: { algorithm: :concurrently }, null: true
```

Comment:
`null: true` looks bad, I would much prefer `false` here. 

Also, the reference is already there, right? All we need is to make it non-nullable now.

Reply:
Ah. May need a rebase. This should be the one from the previous PR.